src/goto-cc/compile.h

```diff
@@ -45,7 +45,8 @@ class compilet:public language_uit
   std::list<irep_idt> seen_modes;
 
   std::string object_file_extension;
-  std::string output_file_object, output_file_executable;
+  std::string output_file_object, output_directory_object;
```

Comment:
Comment that these are mutually exclusive, and are not a pathname / basename pair as they appear

Reply:
done